modules/test/playwright/tests/object-web/objectDefinitions.spec.ts

```diff
@@ -88,4 +90,97 @@ test.describe('Manage object definitions through Model Builder', () => {
 			objectDefinition.id
 		);
 	});
+
+	test('linked object definitions are created when object definitions are related and put into different folders', async ({
+		apiHelpers,
+		modelBuilderPage,
+		page,
+		viewObjectDefinitionsPage,
+	}) => {
+		await page.goto('/');
+
+		const objectFolder =
+			await apiHelpers.objectAdmin.postRandomObjectFolder();
+
+		const objectDefinition1 =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition(
+				objectFolder.externalReferenceCode
+			);
+
+		const objectDefinition2 =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition('default');
+
+		const objectRelationshipLabel =
+			'objectRelationshipLabel' + getRandomInt();
+		const objectRelationshipName =
+			'objectRelationshipName' + objectRelationshipRandomNumber;
+
+		const objectRelationshipData: Partial<ObjectRelationship> = {
+			label: {
+				en_US: objectRelationshipLabel,
+			},
+			name: objectRelationshipName,
+			objectDefinitionExternalReferenceCode1:
+				objectDefinition1.externalReferenceCode,
+			objectDefinitionExternalReferenceCode2:
+				objectDefinition2.externalReferenceCode,
+			objectDefinitionId1: objectDefinition1.id,
+			objectDefinitionId2: objectDefinition2.id,
+			objectDefinitionName2: objectDefinition2.name,
+			type: 'oneToMany' as ObjectRelationshipType,
+		};
+
+		await apiHelpers.objectAdmin.postObjectRelationship(
+			objectRelationshipData
+		);
+
+		await viewObjectDefinitionsPage.goto();
+
+		await viewObjectDefinitionsPage.openObjectFolder('default');
+
+		await viewObjectDefinitionsPage.viewInModelBuilder();
```

Comment:
Hey @rebsilva, we can use model buider go to function here:

`await modelBuilderPage.goto({objectFolderName: 'default'})`

I could sometimes see the page taking a long time to load in some tests, but I'm not sure if it was because of this `goto` function, let me know if you chose to navigate through the buttons because of this.